LPFilter/Source/PluginProcessor.cpp

```diff
@@ -110,6 +113,15 @@ void LpfilterAudioProcessor::prepareToPlay (double sampleRate, int samplesPerBlo
     // Prepare lpfJuce filter
     dsp::ProcessSpec spec {sampleRate, static_cast<uint32>(samplesPerBlock), channels};
     lpfJuce.prepare(spec);
+    
+    // Prepare lpfDspLib filter
+    paramsDsp[0] = sampleRate; paramsDsp[1] = 1; paramsDsp[2] = defaultFreq;
+    lpfDspLib->setParams(paramsDsp);
+    
+    if (getTotalNumInputChannels() < 2 )
```

Comment:
I'm not sure if in `prepareToPlay` the `getTotalNumInputChannels ` returns the correct number of channels. You should remove the if statement and keep only line 123